tests/test_utils.py

```diff
@@ -820,22 +820,24 @@ def batch_size(value):
 @pytest.mark.parametrize(
     "iterable, batch_size_, expected_batches",
     [
-        ([1, 2, 3], batch_size(1), [[1], [2], [3]]),
-        ([1, 2, 3], batch_size(2), [[1, 2], [3]]),
-        (
-            [1, 2, 3],
-            batch_size(3),
-            [
-                [1, 2, 3],
-            ],
-        ),
-        (
-            [1, 2, 3],
-            batch_size(1000),
-            [
-                [1, 2, 3],
-            ],
-        ),
+        # ([1, 2, 3], batch_size(1), [[1], [2], [3]]),
+        # ([1, 2, 3], batch_size(2), [[1, 2], [3]]),
+        # (
+        #     [1, 2, 3],
+        #     batch_size(3),
+        #     [
+        #         [1, 2, 3],
+        #     ],
+        # ),
+        # (
+        #     [1, 2, 3],
+        #     batch_size(1000),
+        #     [
+        #         [1, 2, 3],
+        #     ],
+        # ),
+        # ([], batch_size(20), []),
+        ([[]], batch_size(20), [[[]]]),
```

Comment:
Did you include this change accidentally?

Reply:
The commented out test cases were a mistake 🙈 
I did intend to add these other two test cases, to show that the issue doesn't seem to be a bug in `iterable_batches_generator`